Library/Homebrew/requirements/arch_requirement.rb

```diff
@@ -14,21 +14,47 @@ class ArchRequirement < Requirement
   attr_reader :arch
 
   def initialize(tags)
-    @arch = tags.shift
+    @arch = []
+    tags.each do |tag|
+      case tag
+      when :x86_64, :arm64, :arm, :intel, :ppc
```

Comment:
Bit wary of hardcoding this here. Maybe we can use `Hardware::CPU::ALL_ARCHS`?

Reply:
aye, good point - I didn't notice that constant otherwise I would have totally used it - thanks for the note!

Reply:
Ah, wait, I'm not sure that's what we want, since
```
❯ brew ruby -e 'puts Hardware::CPU::ALL_ARCHS'
i386
x86_64
ppc
ppc32
ppc7400
ppc7450
ppc970
ppc64
ppc64le
ppc970
arm64
```

Reply:
Seems fine to use this to me 👍🏻